code/game/gamemodes/wizard/soulstone.dm

```diff
@@ -206,19 +206,14 @@
 	switch(choice)
 		if("FORCE")
 			var/obj/item/soulstone/C = src
-			if(!iscarbon(target))		//TODO: Add sacrifice stoning for non-organics, just because you have no body doesnt mean you dont have a soul
-				return 0
-			if(contents.len)
-				return 0
-			var/mob/living/carbon/T = target
+			var/mob/living/T = target
 			if(T.client != null)
-				for(var/obj/item/W in T)
-					T.unEquip(W)
 				C.init_shade(T, U)
-				return 1
 			else
 				to_chat(U, "<span class='userdanger'>Capture failed!</span>: The soul has already fled its mortal frame. You attempt to bring it back...")
-				return C.getCultGhost(T,U)
+				T.Paralyse(10)

```

Comment:
Some items and chems will reduce this time. Shouldn't be that much of an issue though since the player is out of the mob.

Reply:
I'll just double the time since the duration doesn't really matter.